tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/NUnit/IMethodInfo.cs

```diff
@@ -2,19 +2,19 @@
 // Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
+#nullable enable
 
 using System.Reflection;
 
-namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.NUnit
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.NUnit;
+
+/// <summary>
+/// DuckTyping interface for NUnit.Framework.Interfaces.IMethodInfo
+/// </summary>
+internal interface IMethodInfo
 {
     /// <summary>
-    /// DuckTyping interface for NUnit.Framework.Interfaces.IMethodInfo
+    /// Gets the MethodInfo for this method.
     /// </summary>
-    internal interface IMethodInfo
-    {
-        /// <summary>
-        /// Gets the MethodInfo for this method.
-        /// </summary>
-        MethodInfo MethodInfo { get; }
-    }
+    MethodInfo MethodInfo { get; }
```

Comment:
Could this be null? Is it safer to assume it could be?

Reply:
done in [3d98a45](https://github.com/DataDog/dd-trace-dotnet/pull/6374/commits/3d98a45a5bebdc0c7de9990b062d3e8eb506713d)